x-pack/platform/plugins/shared/agent_builder_platform/server/tools/product_documentation.ts

```diff
@@ -90,7 +94,7 @@ export const productDocumentationTool = (
         // Return documentation results
         return {
           results: result.documents.map((doc: RetrieveDocumentationResultDoc) => ({
-            type: ToolResultType.resource,
+            type: ToolResultType.other,
```

Comment:
@pgayvallet, I noticed there was a type mis-match here as both these tools were `ToolResultType.resource` but returned `url`/`title`, so I changed them to `ToolResultType.other`. Would you prefer I introduce a new `urlResource` type instead?